File: oscexchange/src/main/java/ac/at/fhstp/digitech/oscexchange/errors/OSCMessageError.java
package ac.at.fhstp.digitech.oscexchange.errors;

import ac.at.fhstp.digitech.oscexchange.OSCArgs;

public final class OSCMessageError extends OSCArgsError {

    public OSCMessageError(Exception exception, OSCArgs args) {
        super(exception, args);
    }

}

File: oscexchange/src/main/java/ac/at/fhstp/digitech/oscexchange/errors/OSCMessageError.kt
package ac.at.fhstp.digitech.oscexchange.errors

import ac.at.fhstp.digitech.oscexchange.OSCArgs

/**
 * An error that occurs when trying to construct an OSC-message to send
 */
class OSCMessageError(exception: Exception, args: OSCArgs) :
    OSCArgsError(exception, args)